runtime/compiler/control/J9Options.cpp

```diff
@@ -1146,6 +1170,55 @@ static bool JITaaSParseOptionsCommon(char **options, char delimiter, TR::Compila
    return true;
    }
 
+static void JITaaSParseEnvVarCommon(TR::CompilationInfo *compInfo, const struct TR_JTIaaSCommonOptions & optionsSet)
```

Comment:
I would like a comment here describing what the function does: checks for env vars related to JITaaS and sets options accordingly. The optionSet specifies which JITaaS options have been already specified using command line options